src/cpu/minor/Ticked.py

```diff
@@ -35,7 +35,9 @@
 
 from m5.objects.ClockedObject import ClockedObject
 
-class TickedObject(ClockedObject):
-    type = 'TickedObject'
+class Ticked():
```

Comment:
This definitely won't work. It's not a SimObject.

Does this need to be a Python object?

Reply:
I don't think that it needs to be  a Python object. Do we need to create  a Python object if it has no associated parameters or it is never going to be accessed by the user through Python interface?

Reply:
Yeah, we can probably drop the python completely.